apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/R4EobSamhsaMatcher.java

```diff
@@ -17,7 +25,16 @@
  * as a singleton.
  */
 @Component
-public final class R4EobSamhsaMatcher extends AbstractR4SamhsaMatcher<ExplanationOfBenefit> {
+public final class R4EobSamhsaMatcher extends AbstractSamhsaMatcher<ExplanationOfBenefit> {
+
+  // Valid system url for productOrService coding
+  private static final Set<String> HCPCS_SYSTEM = Set.of(TransformerConstants.CODING_SYSTEM_HCPCS);
+  // Additional valid coding system URL for backwards-compatibility
+  // See: https://jira.cms.gov/browse/BFD-1345
+  private static final Set<String> BACKWARDS_COMPATIBLE_HCPCS_SYSTEM =
```

Comment:
Do you envision a time whee we do not need to have this 'backwards compatibility'?

Reply:
I'm not sure, this was just code that was moved from another class.  I don't recall the specifics of why this was here originally.

Reply:
@mackec1 this is just used for EOB's, so adjudicated data coming from CCW.   That decision is up to the BFD team.